http/src/test/scala/com.snowplowanalytics.snowplow.storage.kinesis.elasticsearch/clients/ElasticsearchSenderHTTPSpec.scala

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
Maybe tags and branch have different work trees and you pushed only tags recently?